### ISSUE TITLE
[Suggestion] Adding titleFont, subtitleFont and actionTitleFont

### ISSUE THREAD

**Issue by sasojadrovski** [HamzaGhazouani/HGPlaceholders#11]
Hi @HamzaGhazouani,

I am thinking whether adding new properties for setting a custom font for the title, subtitle and action title to the `PlaceholderStyle` struct is something you have considered also?

I find that this will be helpful to a lot of developers that are going to be using this library.

Any thoughts?

Cheers!

**Comment by HamzaGhazouani** [comment]
Hi @sasojadrovski,
I think it's a good idea, we will have a lot of properties in the `PlaceholderStyle` struct but it's not a big deal, on the other hand, we will have more options for customization

**Comment by junweimah** [comment]
I have a simple question, sorry for being a ios newbie.

How do I change the title and sub title wording as I am displaying the placeholders? 

I found out that the wordings are in `PlaceholderData.swift`, in `public struct PlaceholderData`. 

Lets say I do not wish to hard code them here, but set it to something when I am calling `tableView.showNoConnectionPlaceholder()`, how do I do it?

I also found the file `PlaceholdersProvider.swift`, which I set it to be : 

```
var noConnectionData: PlaceholderData = .noConnection
        noConnectionData.image = #imageLiteral(resourceName: "hg_default-error")
        noConnectionData.title = "Testing no connection title"
        noConnectionData.subtitle = "Testing no connection sub title"
        let noConnection = Placeholder(data: noConnectionData, style: commonStyle, key: .noConnectionKey)
```

But the message is still the same

How do I do it?

Thanks

**Comment by HamzaGhazouani** [comment]
Hi @junweimah, can you please create a new issue with your question ?

**Comment by HamzaGhazouani** [comment]
Available in [HGPlaceholders 0.3.0](https://github.com/HamzaGhazouani/HGPlaceholders/releases/tag/0.3.0)
